### PR TITLE
libmupdf: new package at 1.16.1

### PR DESCRIPTION
**Pull request by ioraff**
Read: <getkiss.org/guidestones.txt>

## Description of package
mupdf libs and command line tools

## New package

- [x] Latest upstream version.
- [x] I agree to maintain this package.
- [x] I agree to notify @dylanaraps if I can no longer maintain this package.

## Existing package

- [ ] I am the maintainer of this package.

### CODE REVIEW

File: community/libmupdf/build [hunk 1]
```diff
@@ -0,0 +1,13 @@
+#!/bin/sh -e
+
+patch -p1 < pkcs7-fix.patch
+
+export CFLAGS="$CFLAGS -fPIC"
+export CXXFLAGS="$CXXFLAGS -fPIC"
```
Comment (dylanaraps) [inline community/libmupdf/build]: Needed?

File: community/libmupdf/build [hunk 2]
```diff
@@ -0,0 +1,13 @@
+#!/bin/sh -e
+
+patch -p1 < pkcs7-fix.patch
```
Comment (dylanaraps) [inline community/libmupdf/build]: What does the patch do?

Comment (ioraff) [reply community/libmupdf/build]: I believe it's a libressl fix. Build fails without it.

File: community/libmupdf/build (hunk 1 above)
Comment (ioraff) [reply community/libmupdf/build]: Not absolutely

**Pull request merged**